recipes/fermipy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.10.4" %}
+
+package:
+  name: fermipy
+  version: {{version}}
+
+source:
+  fn: fermipy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/f/fermipy/fermipy-{{ version }}.tar.gz 
+  md5: 3eee87e3d2f324fdc51afe03f9d086a5
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
```

Comment:
The package depends on a number of external libraries which are only available on OSX and Linux.  The user base for this package are astronomers and physicists who almost exclusively use OSX or Linux so I think the demand for a Windows build would be pretty low.